docker-compose.yml

```diff
@@ -31,6 +31,31 @@ services:
       HASURA_GRAPHQL_AUTH_HOOK: http://host.docker.internal:3000/api/hasura/auth
       ## HASURA_GRAPHQL_UNAUTHORIZED_ROLE: public
 
+
+  # for local s3 storage for images
+  localstack:
+    image: localstack/localstack:latest
+    hostname: s3
+    networks:
+      default:
+        aliases:
+          - s3.localhost.localstack.cloud
+          - s3-website.localhost.localstack.cloud
+          - coordinape.s3.localhost.localstack.cloud
+          - coordinape.s3-website.localhost.localstack.cloud
+    environment:
+      - HOSTNAME_EXTERNAL=localstack
+      - AWS_DEFAULT_REGION=us-east-1
+      - EDGE_PORT=4566
```

Comment:
having the port as a .env variable seems good, with the other variables

Reply:
None of the other ports in this file have vars for them, how do you want to do it? There is a range for localstack but i dont think its necessary, we probably only need 4566.

Reply:
Never mind i see it in main, i'll sort this out

Reply:
this is all that is needed to be ready to merge, we can do more for improving env variables in another PR

@levity has feelings about the env as well

Reply:
eh, don't want to complexify the env, if it comes up, we can address it then